File: src/typescript/src/transport/in-memory/__tests__/alpha-command.ts
import { messageInfo } from '../../transaction/message/message-info';
import { MessageType } from '../../transaction/message/message-type';

/**
 * Test command message for alpha endpoint
 */
@messageInfo(MessageType.Command, 'alpha', 'alpha-command', 1, 0, 0)
export class AlphaCommand {
  public name!: string;
}

File: src/kotlin/src/test/kotlin/polybus/transport/inmemory/AlphaCommand.kt
package polybus.transport.inmemory

import polybus.transport.transactions.messages.MessageInfo
import polybus.transport.transactions.messages.MessageType

@MessageInfo(MessageType.COMMAND, "alpha", "alpha-command", 1, 0, 0)
class AlphaCommand {
    lateinit var name: String
}
